templates/static-fileserver/metadata/spin-template.toml

```diff
@@ -5,12 +5,12 @@ trigger_type = "http"
 
 [add_component]
 skip_files = ["spin.toml"]
-skip_parameters = ["http-base", "project-description"]
+skip_parameters = ["http-base", "application-description"]
 [add_component.snippets]
 component = "component.txt"
 
 [parameters]
-project-description = { type = "string",  prompt = "Project description", default = "" }
+application-description = { type = "string",  prompt = "Application description", default = "" }
```

Comment:
Now that I'm looking at this, I agree with Ivan's suggestion for letting the prompt be just "Description". What do you think @calebschoepp ?

Reply:
Yep think that's great. Changing it right now.